cpp/CMakeLists.txt

```diff
@@ -2,30 +2,35 @@ cmake_minimum_required( VERSION 3.5.1 )
 
 project(lab_00_intro)
 
-# Compiler flags
-if (MSVC)
-  set(compiler_options /W4)
-  add_definitions(/D_USE_MATH_DEFINES)
-else ()
-  set(compiler_options -Wall -Wextra -Wpedantic)
-endif (MSVC)
+list(PREPEND CMAKE_MODULE_PATH ${CMAKE_BINARY_DIR})
 
 # Find libraries
 find_package(OpenCV 4 REQUIRED)
-find_package(Eigen3 REQUIRED)
-find_package(Sophus REQUIRED)
-find_package(GeographicLib REQUIRED)
 
-add_executable(${PROJECT_NAME} main.cpp)
+set(exe_name lab_00_intro)
+add_executable(${exe_name} main.cpp)
 
-target_link_libraries(${PROJECT_NAME}
+if (NOT OpenCV_LIBS)  # If conan
+  set(OpenCV_LIBS "opencv::opencv")
+endif()
+
+target_link_libraries(${exe_name}
```

Comment:
Jeg gikk for denne løsningen for å beholde bakoverkompatibitet med ikke-conan, det vil si dem som allerede baserer seg på `setup_scripts`

Mulige alternativer:

1. Bare ha `target_link_libraries(opencv::opencv)` og si "installer conan :middle_finger: 
2. Ha det sånn som her, men kanskje støte på andre uforutsette hindere senere pga forsøk på å støtte med og uten conan.

Reply:
Veldig bra, vi kan jo vurdere å pynte til neste år